### PR TITLE
Ndev 652 Run UI tests via github actions

### PR DESCRIPTION
**Pull request by lyubenkov**
Reminder!
Before merge please change docker tag to `latest` in playwright.yml.

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+on:
+  schedule:
+    - cron: "0 1 * * 5"
+  workflow_dispatch:
+jobs:
+  playwright:
+    timeout-minutes: 60
+    runs-on: ubuntu-22.04
+    env:
+      CHROME_EXT_PASSWORD: "1234Neon5678"
```
Comment (gigimon) [inline .github/workflows/playwright.yml]: please move this password to secrets

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - name: "Generate allure report"
+        if: always()
+        uses: ./.github/actions/generate-allure-report
+        with:
+          container: ${{ env.CONTAINER }}
+          network: ${{ env.NETWORK }}
+          aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          tests_name: ui
+      - name: "Remove Docker Container"
```
Comment (gigimon) [inline .github/workflows/playwright.yml]: we don't need this, because github destroy runner after run

**Review by gigimon — APPROVED** [review]

File: .github/workflows/playwright.yml (hunk 2 above)
Comment (lyubenkov) [reply .github/workflows/playwright.yml]: Fixed

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (lyubenkov) [reply .github/workflows/playwright.yml]: Fixed

File: .github/workflows/playwright.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      - uses: actions/checkout@v3
+      - name: Install python requirements
+        id: requirements
+        uses: ./.github/actions/python-requirements
+      - name: Notify on failure
+        id: notification
+        run: |
+          ./clickfile.py send-notification \
+          -u ${{ secrets.SLACK_QA_CHANNEL_URL }} \
+          -b ${{ env.BUILD_URL }} \
```
Comment (kristinaNikolaevaa) [inline .github/workflows/playwright.yml:57]: ${{ env.BUILD_URL }} is not defined

**Review by kristinaNikolaevaa — CHANGES_REQUESTED** [review]

File: .github/workflows/playwright.yml (hunk 3 above)
Comment (lyubenkov) [reply .github/workflows/playwright.yml:57]: Good point, added/fixed!

File: .github/workflows/playwright.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+  workflow_dispatch:
+jobs:
+  playwright:
+    timeout-minutes: 60
+    runs-on: ubuntu-22.04
+    env:
+      NETWORK: "devnet"
+      IMAGE: neonlabsorg/neon_tests:ebd371e6341c088b04b8cf41cde0e9d91804874c
```
Comment (gigimon) [inline .github/workflows/playwright.yml:15]: Please use latest image (without hash)

**Review by gigimon — APPROVED** [review]

**Pull request merged**